src/main/java/org/sonarlint/intellij/config/global/SonarLintGlobalSettings.java

```diff
@@ -182,6 +182,10 @@ public Optional<ServerConnection> getServerConnectionByName(String name) {
       .findFirst();
   }
 
+  public boolean exists(String connectionName) {
```

Comment:
We can use the name serverConnectionExists or connectionExist instead of exists

Reply:
:+1: